test-case/manual/check-simple-probe-capture.md

```diff
@@ -0,0 +1,77 @@
+# Probes test
+Simple probes functionality test.
+
+# Preconditions
+1. Linux kernel should have these config options enabled:
+   CONFIG_SND_SOC_SOF_DEBUG_PROBES=y
+   CONFIG_SND_SOC_SOF_HDA_PROBES=y
+
+2. Firmware should have these config options enabled:
+   CONFIG_PROBE=y
+   CONFIG_PROBE_POINTS_MAX=16
+   CONFIG_PROBE_DMA_MAX=4
```

Comment:
The PROBE_DMA_MAX is for injection which is not supported on Linux and I dare to say that it has never been tested?
I would not mention it for now. The extraction works just fine with the default value of 0.